flow360/component/simulation/mesh.py

```diff
@@ -0,0 +1,47 @@
+from typing import List, Optional, Union
+
+import pydantic as pd
+
+from flow360.component.simulation.framework.base_model import Flow360BaseModel
+
+from .zones import BoxZone, CylindricalZone
+
+
+class FaceRefinement(Flow360BaseModel):
```

Comment:
Should we names consistent with what are used in the simulation interface?

SurfaceRefinement? VolumeRefinement?

Reply:
We do have this Face concept right now which is different from Surface.

`Surface` is defined as the group of patches (which is the smallest unit) that share the same **boundary condition**. Face is defined as the group of patches (which is the smallest unit) that share the same **meshing parameters**

Reply:
I don't think there should be such a distinction. A group of patches should be one entity. We can either use it for meshing or solver or both. Why are you making the user confused that it is called different things in different places

Similar for ZoneRefinement. Why not just call it VolumeRefinement

Reply:
This is not something coming from Simulation design. The Face-Surface distinction comes from our mesher. I also in favor of unification. CC: @rundaji

Reply:
Just FYI, the patches (NURBS) can be categized in two different ways
(1) `faceName` v.s. (2) `groupName`/`boundaryName`

(1) `faceName` is closely related to meshing:
As shown in figure 3.3.2, the patches with same `faceName` will share the same max_edge_length
https://docs.flexcompute.com/projects/flow360/en/latest/preprocessing/automatedMeshing/automatedMeshing.html#surface-meshing
As shown in figure 7.7.2, the patchces with same `faceName` will share the same
https://docs.flexcompute.com/projects/flow360/en/latest/tutorials/AutoMeshingInternalFlow/AutoMeshingInternalFlow.html#volume-mesh

(2) `groupName`/`boundaryName` is more related to boundary conditions when dumping the ugrid/cgns files

Reply:
@benflexcompute Could you direct me to where `Surface` is define for group of patches for boundary conditions?